packages/flame/lib/src/effects/effects.dart

```diff
@@ -243,40 +381,19 @@ abstract class PositionComponentEffect
   }
 
   @override
-  void setComponentToEndState() {
-    _setComponentState(endPosition, endAngle, endSize, endScale);
+  void setComponentToPeakState() {
+    _setComponentState(peakPosition, peakAngle, peakSize, peakScale);
   }
 }
 
-abstract class SimplePositionComponentEffect extends PositionComponentEffect {
-  double? duration;
-  double? speed;
+mixin EffectsHelper on Component {
```

Comment:
Ah, found it 😅 , this file had too many changes, GitHub collapsed it.

What if we call this `EffectsHandlers` instead of `EffectsHelper`?

Reply:
or HasEffects ?

Reply:
`EffectsHandlers` sounds like it would do something, but it is just two helper methods, you don't need this mixin to use effects if you don't want to. `HasEffects` sounds like it would be similar to `HasDraggables/Tappables` etc, but it is not.

Reply:
On a second thought, since it is only two methods couldn't we just add them to components directly?

Reply:
Hmm, I thought we didn't want a lot of extra methods on there?
But it would probably make it easier for the user in the end, so I'd be for it.

Reply:
Usually I wouldn't want too, but idk, Helper extensions seems weird to me, but I have no logical good reason for this, is just a personal thing. So I leave it up to you.